cognite/client/_cognite_client.py

```diff
@@ -83,6 +88,11 @@ def __init__(self, config: ClientConfig | None = None) -> None:
         self.documents = DocumentsAPI(self._config, self._API_VERSION, self)
         self.workflows = WorkflowAPI(self._config, self._API_VERSION, self)
         self.units = UnitAPI(self._config, self._API_VERSION, self)
+        self.simulators = SimulatorsAPI(self._config, self._API_VERSION, self)
+        self.simulators.models = SimulatorModelsAPI(self._config, self._API_VERSION, self)
+        self.simulators.runs = SimulatorRunsAPI(self._config, self._API_VERSION, self)
+        self.simulators.integrations = SimulatorIntegrationsAPI(self._config, self._API_VERSION, self)
+        self.simulators.routines = SimulatorRoutinesAPI(self._config, self._API_VERSION, self)
```

Comment:
they are part of `SimulatorsAPI` constructor, right? so no need to reinitiate here?